schemas/com.snowplowanalytics.snowplow/identify/jsonschema/1-0-0

```diff
@@ -0,0 +1,25 @@
+{
+        "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+        "self": {
+                "vendor": "com.snowplowanalytics.snowplow",
+                "name": "identify",
+                "version": "1-0-0",
+                "format": "jsonschema"
+        },
+        "description": "A Snowplow identify event.",
+        "type": "object",
+        "properties": {
+                "id": {
+                        "description": "The user identifier.",
+                        "type": ["string", "null"],
+                        "maxLength": 256
+                },
+                "email": {
+                        "description": "The user email address.",
+                        "type": ["string", "null"],
+                        "maxLength": 256
+                }
```

Comment:
I assume both `id` and `email` may be null because someone may send one *or* the other, but in that case isn't `email` an `id` really? In which case, you'd only need a non-null `id` and the value of it would be an email address.

Reply:
I'd be inclinded to agree with you, however many platforms split out these two fields when we pass this data to them and they behave differently depending on if you give them an email address or an id. With marketing platforms, once you give them an e-mail address they start to build a user profile, whereas because an id could be an email or could be a uuid they don't make the same assumptions. Iterable is a good example, where you can pass an e-mail and/or a userId: https://api.iterable.com/api/docs#events_track